app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+
+  def update_number_of_comments
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] Great work on defining the methods for your comment model! 😊 To enhance the maintainability and organization of your code, I recommend considering the use of [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), specifically `after_save`, to automatically trigger the `update_number_of_comments ` method each time a new comment is saved to the database. This approach will help you keep your code clean and ensure that the counter is always up to date without manual intervention.
   - Additionally, to encapsulate the `update_number_of_comments` method properly and prevent accidental external calls, consider marking it as private. This encapsulation will improve code readability and adherence to best practices.

Happy coding! 👍👨‍💻🚀